sdk/beatoven_sdk/client.py

```diff
@@ -0,0 +1,80 @@
+from .api import get_track, list_tracks, create_track, compose_track, get_task_status
+import asyncio
+
+
+def extract_track_info(track):
+    track_info = {}
+    track_info["title"] = track["title"]
+    track_info["id"] = track["project_id"]
+    return track_info
+
+
+async def list_all_tracks():
+    tracks_data = await list_tracks()
+    return list(map(extract_track_info, tracks_data["tracks"]))
+
+
+async def get_track_by_id(track_id):
+    track_data = await get_track(track_id)
+    track_url = track_data["download_url"]
+    return track_id, track_url
+
+
+async def create_new_track(
```

Comment:
since this function anyway handles composition, wdyt about renaming this to compose track ?

Reply:
Yeah that would make sense, will update